server/mdm/apple/vpp/api.go

```diff
@@ -33,7 +33,7 @@ type Asset struct {
 //
 // https://developer.apple.com/documentation/devicemanagement/errorresponse
 type ErrorResponse struct {
-	ErrorInfo    ResponseErrorInfo `json:"errorInfo"`
+	ErrorInfo    ResponseErrorInfo `json:"errorInfo,omitempty"`
```

Comment:
Curious: why omitempty here?

Reply:
Apple doesn't return an error object so it makes the mock api more realistic